### PR TITLE
Use language parameter

### LINKED ISSUE #5: Hotels query needs to support language parameter
We need to add language parameter to get hotel summary in the user language

### PR DESCRIPTION
**Pull request by tbergq**
closes #5

### CODE REVIEW

File: src/hotel/queries/Hotel.js [hunk 1]
```diff
@@ -14,10 +14,13 @@ export default {
     id: {
       type: new GraphQLNonNull(GraphQLID),
     },
+    language: {
+      type: GraphQLString,
```
Comment (mrtnzlml) [inline src/hotel/queries/Hotel.js]: What about using ENUM here instead of just string?

File: src/hotel/queries/__tests__/Hotel.fullQuery.test.js [hunk 2]
```diff
@@ -13,6 +13,9 @@ it('works with full query', async () => {
   BookingComApiMock.onGet(
     'https://distribution-xml.booking.com/2.0/json/hotels?extras=hotel_info%2Chotel_photos%2Chotel_description%2Chotel_facilities%2Cpayment_details%2Croom_info%2Croom_photos%2Croom_description%2Croom_facilities&hotel_ids=25215',
   ).replyWithData(SingleHotelDataset);
+  BookingComApiMock.onGet(
+    'https://distribution-xml.booking.com/2.0/json/hotels?extras=hotel_info%2Chotel_photos%2Chotel_description%2Chotel_facilities%2Cpayment_details%2Croom_info%2Croom_photos%2Croom_description%2Croom_facilities&hotel_ids=25215%2C25215',
+  ).replyWithData(SingleHotelDataset);
```
Comment (mrtnzlml) [inline src/hotel/queries/__tests__/Hotel.fullQuery.test.js]: Why is this change here?

**Comment by michalsanger** [comment]
`availableHotel` query is missing language parameter
